modules/tensor_mechanics/src/materials/RadialReturnStressUpdate.C

```diff
@@ -277,6 +296,9 @@ RadialReturnStressUpdateTempl<false>::updateStateSubstep(RankTwoTensor & strain_
                                                          RankFourTensor & tangent_operator)
 {
   const unsigned int total_substeps = calculateNumberSubsteps(strain_increment);
+  _number_of_substeps[_qp] = total_substeps;
+
+  Moose::out << "Number of substeps: " << total_substeps << "\n";
```

Comment:
This info will be printed our for all `_qp`, right? Maybe this is too much info? Are we going to store this in an additional material property?

Reply:
Definitely too much information! That must go!

Reply:
Yeah, the one test should be outputting the number of steps to the exodus file.